common/display/displayqueue.cpp

```diff
@@ -689,7 +701,7 @@ void DisplayQueue::HandleExit() {
     cloned_mode = true;
   }
 
-  state_ = kConfigurationChanged;
+  state_ = kConfigurationChanged | (state_ & kApplyColorMatrix);
```

Comment:
We shouldn't always set this. Like for  disable_overlay below, we should check if this is part of state_ already before setting it again.